button-react-app/src/App.tsx

```diff
@@ -31,4 +30,4 @@ function App() {
   )
 }
 
-export default App
+export default App
```

Comment:
Please, config your [git auto crlf](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#global-settings-for-line-endings) in your machine